setup/postgresql_bootstrap_centos7.sh

```diff
@@ -110,6 +110,9 @@ END_OF_HBA_CONF"
 
   # Log message is produced when a session waits longer than deadlock_timeout to acquire a lock.
   sudo sed -i -e "s/.*log_lock_waits.*/log_lock_waits = on/" ${PGDATA}/postgresql.conf
+  
+  # Set a default statement timeout to 5 minutes
+  sudo sed -i -e "s/.*statement_timeout.*/statement_timeout = 5min/" ${PGDATA}/postgresql.conf
```

Comment:
should be `= '5min'`

Quotes are required when using units: https://www.postgresql.org/docs/9.6/static/config-setting.html

Reply:
Interesting. They probably do not follow their own requirements. :-)
<img width="530" alt="screen shot 2018-03-07 at 19 04 03" src="https://user-images.githubusercontent.com/13724127/37076207-754281da-223a-11e8-986c-87144e1ee76d.png">
<img width="572" alt="screen shot 2018-03-07 at 18 59 25" src="https://user-images.githubusercontent.com/13724127/37076209-7aac34b8-223a-11e8-8fbc-09bc6348822f.png">

Reply:
doc is probably to up-to-date, feel free to commit as is.